packages/env/lib/detect-context.js

```diff
@@ -13,21 +13,45 @@ const path = require( 'path' );
 const readDir = util.promisify( fs.readdir );
 const finished = util.promisify( stream.finished );
 
-module.exports = async function detectContext() {
+const cwd = process.cwd();
+
+/**
+ * @typedef Context
+ * @type {Object}
+ * @property {string} type
+ * @property {string} path
+ * @property {string} pathName
```

Comment:
```suggestion
 * @property {string} pathBasename
```

`cwdName` made sense, because it was the name of the current working directory, but here it's not the path's name, it's the path's basename.

Reply:
Good call